app/src/main/java/com/github/se/polyfit/model/post/MapLocationModel.kt

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
You can delete this file

Reply:
Yes mb on this file, but I need to work on it later for the location for the map this time, because this is different and it will be the `model` for the location map, and I've already written some part of code that I will need and still work on this branch after the merge, if it's okay to let it

Reply:
:+1: